### ISSUE TITLE
ESL 5.4

### ISSUE THREAD

**Issue by HanXudong** [talwarabhimanyu/my-solutions-The-Elements-of-Statistical-Learning#1]
Hey Abhimanyu,

Your solutions do help me! Thanks!
For chapter 5, equation 60, I think there should be a $\theat_k$ after the summation symbol.

**Comment by talwarabhimanyu** [comment]
Yes, that's indeed true. Thanks for catching that! I'll close this issue after I update the file. Cheers
